pkg/cvo/cvo.go

```diff
@@ -195,7 +195,7 @@ func New(
 		kubeClient:    kubeClient,
 		eventRecorder: eventBroadcaster.NewRecorder(scheme.Scheme, corev1.EventSource{Component: namespace}),
 
-		queue:                 workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "clusterversion"),
+		queue: workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "clusterversion"),
```

Comment:
What version of Go's `fmt` is removing this indent?

Reply:
> What version of Go's `fmt` is removing this indent?

1.13.4

Reply:
can we wash it through 1.14, to avoid making this churn that will be reverted later?

Reply:
What version is this repo building with?  It should use fmt rules from that go version.

Reply:
if it's not being built with 1.14 now, I will file PRs to pivot to 1.14 so we don't backslide to 1.13 formatting ;)

Reply:
Bumping our local Dockerfiles to Go 1.14: openshift/cluster-version-operator#414 .  Matching release-config pivot (adding a `gofmt` job for master) is openshift/release#10410.

Reply:
Back to 1.14 formatting with 041e96e -> e1de651.  Marking this thread resolved.